dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/tooling/Instrumenter.java

```
@@ -86,7 +86,7 @@
     private final String instrumentationPrimaryName;
     private InstrumentationContextProvider contextProvider;
     private boolean initialized;
-    private final boolean enabled;
+    protected final boolean defaultEnabled;
```

I'm not sure about this change because it replaces something that was clear (`enabled`) with something less clear - ie. `defaultEnabled` is not the same as `defaultEnabled()` and is not actually the "default" , but a combination of the default with a configuration override.

AFAICT this change is just so the scala instrumentations can see if the instrumentation is enabled from the POV of the base class - is there a reason they can't use `super.enabled()` to do that?

---

Sure, I can change it back to `enabled`. I can see how confusing `defaultEnabled` is, even though `enabled` is a bit misleading as well.